roles/cloud-bootstrap/tasks/main.yml

```diff
@@ -3,3 +3,8 @@
   include: network.yml
   tags:
     - network
+
+- name: Provision users and projects
+  include: create-users-projects.yml
+  tags:
+    - create-users-projects
```

Comment:
since the previous tag in this role is more simplistic, `network`, we could be simple in this tag as well, `accounts`. or some such.

However, since the network bits require the users/projects in place, this new task set to create the users and projects needs to run before the network tasks.